pyrobosim/examples/demo.py

```diff
@@ -75,7 +75,7 @@ def create_world(multirobot=False):
     table = world.add_location(
         category="table",
         parent="kitchen",
-        pose=Pose(x=0.85, y=-0.5, z=0.0, yaw=-np.pi / 2.0),
+        pose=Pose(x=0.85, y=-0.5, z=0.0, yaw=-90, angle_units="degrees"),
```

Comment:
Personal preference: please use decimal points for floats, so `90.0`.

Same for all other examples and world files modified

Reply:
Noted.